modules/ROOT/pages/to-deploy-using-rest-connect.adoc

```diff
@@ -412,31 +443,43 @@ securitySchemes:
+Most of the fields are auto-populated based in the GitHub API specification.
 +
-image::rest-connect-api-config.png[]
+image::rest-connect-api-config.png["Configuration window for the GitHub API Configuration component"]
 +
 . Get the Client ID and Client Secret for your GitHub Account. You can find your Client ID and Client Secret if you go to Settings > Developer settings in GitHub. If you don’t have an OAuth App in GitHub, you can  create one with the New OAuth App.
 . Because GitHub API’s base URL is api.github.com, you can put “/” in the Base Path.
-. Match and modify your external callback URL. The callback URL receives an access token from GitHub. By default, the connector shows `+http://my-app.cloudhub.io/callback+`, but you need to modify it specific to your app. The demo app’s callback URL should be `+http://githubapp-smky.cloudhub.io/callback+` so I need to replace “my-app” with “githubapp-smky.” You can find this information to go to the menu and select the copy link in Design Center.
+. Match and modify your external callback URL. The callback URL receives an access token from GitHub. 
++
+By default, the connector shows `+http://my-app.cloudhub.io/callback+`, but you need to modify it to be specific to your app. The demo app’s callback URL is `+http://githubapp-smky.cloudhub.io/callback+`, so replace “my-app” with “githubapp-smky.” You can find this information to go to the menu and select the copy link in Design Center.
 +
 image::rest-connect-ready-to-deploy.png[]
 +
 . After you get your external callback URL, specify the same URL in your GitHub settings.
 +
 image::rest-connect-auth-callback-url.png[]
 +
-. You are ready to retrieve an access token from GitHub. In this case, go to `+http://githubapp-smky.cloudhub.io/authorize+` in a browser, your case would be `+http://my-app.cloudhub.io/authorize+` - replace my-app.cloudhub.io with the one you get with Copy link. When you reach this URL, your browser asks you to log into GitHub.
-. When your access token is issued properly, you can get issues related to Salesforce from GitHub by using `+http://my-app.cloudhub.io/getIssues+` - my-app.cloudhub.io should be replaced with the one you get with `Copy link`.
+You are ready to retrieve an access token from GitHub. 
++
+. Go to `+http://githubapp-smky.cloudhub.io/authorize+` in a browser, for example `+http://my-app.cloudhub.io/authorize+`.
+. Replace `my-app.cloudhub.io` with the one you got when you copied the link. 
+. Log in to GitHub.
++
+When your access token is issued properly, you can get issues related to Salesforce from GitHub by using `+http://my-app.cloudhub.io/getIssues+` - my-app.cloudhub.io should be replaced with the one you get with `Copy link`.
+////
+
+== Known Limitations
 
-== Known limitations
-  - Anypoint platform, including REST Connect, does not support API fragments for OAS.
+  - Anypoint Platform, including REST Connect, does not support API fragments for OAS.
```

Comment:
- TLS support can only be enabled for Mule 4 (SmartConnector) connectors. This feature is not supported for Mule 3 (DevKit).